src/components/atoms/SearchInfoPopup/SearchInfoPopup.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { Card, CardContent, IconButton, Typography } from '@mui/material';
+import Zoom from '@mui/material/Zoom';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+
+export default function SearchInfoPopup() {
+  const [popupOpen, setPopupOpen] = useState(true);
+
+  const handlePopupToggle = () => {
+    setPopupOpen(!popupOpen);
+  };
+
+  return (
+    <Zoom in={popupOpen}>
+      <Card
+        variant="outlined"
+        sx={{
+          position: 'fixed',
+          top: 157,
+          right: 16,
+          zIndex: 999,
```

Comment:
Is a z-index of 999 really necessary? It doesn't look like there are any other elements nearby competing for visibility so a lower number value should be fine here.

Reply:
I may just leave it off because setting the position value already puts it above other elements.

Reply:
Gah, that is kind of annoying. Where what nearby element is using that class?

Reply:
The map is using it.